src/components/EditorPage.js

```diff
@@ -173,7 +174,7 @@ class EditorPage extends Component {
             icon='ion-android-clipboard'
             title='Copy'>
 
-      <CopyToClipboard text={this.props.template.get('html')}>
+            <CopyToClipboard text={beautify.html(this.props.template.get('html'))}>
```

Comment:
That's a good idea. I'll change that so it only updates when it changes like above.